src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts

```diff
@@ -510,17 +532,17 @@ export class CustomThemeComponent implements OnInit, OnChanges, OnDestroy {
         // Check for duplicate theme names and labels
         let dupeName = this.isDuplicateOf("name",data.name);
         if(dupeName){
-          messages.push("This theme name is already taken. Please choose another name.")
+          messages.push("This theme name is in use. Enter another name.")
```

Comment:
"This name is already being used by an existing theme. Please choose a new name."